wfe/web-front-end_test.go

```diff
@@ -211,6 +213,30 @@ func (sa *MockSA) GetCertificateByShortSerial(serial string) (core.Certificate,
 	return sa.GetCertificate("0000000000000000" + serial)
 }
 
+func (sa *MockSA) GetCertificateByRequestID(id string) (core.Certificate, error) {
+	var err error
+	cert := core.Certificate{
+		RegistrationID: 1,
+		RequestID:      id,
+	}
+	switch id {
+	case "weird-weird-weird-weird-weird-weird-weird-x":
+		cert.Status = core.StatusUnknown
+	case "pending-pending-pending-pending-pending-xxx":
+		cert.Status = core.StatusPending
+	case "bad-bad-bad-bad-bad-bad-bad-bad-bad-bad-xxx":
+		cert.Status = core.StatusInvalid
+	case "good-good-good-good-good-good-good-good-xxx":
+		certPemBytes, _ := ioutil.ReadFile("test/178.crt")
```

Comment:
Still don't like that we have these `ioutil.ReadFile()` calls, but that's a separate bug.